sdk/core/azure-core/CHANGELOG.md

```diff
@@ -4,12 +4,33 @@
 
 ### Features Added
 
+- Added native OpenTelemetry tracing to Azure Core which enables users to use OpenTelemetry to trace Azure SDK operations without needing to install a plugin. #39563
```

Comment:
is a note needed here for what steps someone who has installed the plug in needs to take?

Reply:
It'd probably be handy. I'll clarify here.